parlai/mturk/tasks/turn_annotations/launch.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from parlai.mturk.tasks.turn_annotations.run import run_task
+from parlai.core.params import ParlaiParser
+
+
+def launch():
+    """
+    Convenience function to avoid launching from the command line all the time.
+    """
+
+    argparser = ParlaiParser(False, False)
+    datapath = os.path.join(argparser.parlai_home, 'data')
+    task_folder = 'turn_annotations'
+    models_needed_dict = {'TODO_FIXME_MODEL_NAME': 110}
+    override_opt = {
+        'block_qualification': 'block_qualification_name',
+        'base_save_folder': os.path.join(datapath, task_folder),
+        'onboard_worker_answer_folder': os.path.join(
+            datapath, task_folder, 'onboard_answers'
+        ),
+        'base_model_folder': 'TODO_FIXME_BASE_MODEL_FOLDER',
+        'num_conversations': 110,
```

Comment:
should this be `sum(models_needed_dict.valuse())`?

Reply:
Yes that's reasonable.